src/logic/currencyValues/store/actions/setSelectedCurrency.ts

```diff
@@ -1,9 +1,13 @@
 import { createAction } from 'redux-actions'
+import { Currency } from '../model/currencyValues'
 
 export const SET_CURRENT_CURRENCY = 'SET_CURRENT_CURRENCY'
 
 // eslint-disable-next-line max-len
```

Comment:
this eslint-disable rule can be removed with the new formatting

Reply:
done